### ISSUE TITLE
Getting error with Pytorch lightning when passing model checkpoint

### ISSUE THREAD

**Issue by pratikchhapolika** [PyTorchLightning/pytorch-lightning#9488]
I am training a multi-label classification problem using Hugging face models.

I am using `Pytorch lightning` to train the model.
 

Here is the code:

And early stopping triggers when the loss hasn't improved for the last 

    early_stopping_callback = EarlyStopping(monitor='val_loss', patience=2)

We can start the training process:

    checkpoint_callback = ModelCheckpoint(
      dirpath="checkpoints",
      filename="best-checkpoint",
      save_top_k=1,
      verbose=True,
      monitor="val_loss",
      mode="min"
    )

    trainer = pl.Trainer(
      logger=logger,
      callbacks=[early_stopping_callback],
      max_epochs=N_EPOCHS,
     checkpoint_callback=checkpoint_callback,
      gpus=1,
      progress_bar_refresh_rate=30
    )
    # checkpoint_callback=checkpoint_callback,

As soon as I run this, I get error:

    ~/.local/lib/python3.6/site-packages/pytorch_lightning/trainer/connectors/callback_connector.py in _configure_checkpoint_callbacks(self, checkpoint_callback)
         75             if isinstance(checkpoint_callback, Callback):
         76                 error_msg += " Pass callback instances to the `callbacks` argument in the Trainer constructor instead."
    ---> 77             raise MisconfigurationException(error_msg)
         78         if self._trainer_has_checkpoint_callbacks() and checkpoint_callback is False:
         79             raise MisconfigurationException(
    
    MisconfigurationException: Invalid type provided for checkpoint_callback: Expected bool but received <class 'pytorch_lightning.callbacks.model_checkpoint.ModelCheckpoint'>. Pass callback instances to the `callbacks` argument in the Trainer constructor instead.

**How can I fix this issue?**

**Comment by pratikchhapolika** [comment]
> `checkpoint_callback` supports only a bool value. If set to True, it will create a model checkpoint instance internally, but if you want to assign your own custom instance then pass it within callbacks:
> 
> ```python
> trainer = Trainer(callbacks=[checkpoint_callback, early_stopping_callback], ...)
> ```

Thanks. Closing.
